plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -50,16 +50,20 @@ function getCommandFromText(text?: string): string {
   return words[0].length > 0 ? words[0] : 'unknown';
 }
 
-export function getJDBCString(params: PgClientConnectionParams) {
```

Comment:
Removed this change and reused the same type albeit with changing the fields to possibly be undefined. In `client.connect` case the client itself is passed to `getConnectionString` and the client has all of these fields as optional. Perhaps the `PgClientConnectionParams` fields should've been optional since the beginning.